### PR TITLE
Adding docs about generating metadata feature in integration studio

### PR DESCRIPTION
**Pull request by sajithaliyanage**
Doc of Fix: https://github.com/wso2/integration-studio/issues/888

### CODE REVIEW

File: en/docs/tutorials/integration-tutorials/service-catalog-tutorial-for-proxy-services.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ Also, this demonstrates how the integration components and API management compon
 ## Let's get started!
 
 Follow the steps given below to build this use case and try it out.
+    !!! Note
+        See the [Generating Service Catalog Metadata Artifact]({{base_path}}/integrate/develop/generate-service-catalog-metadata.md) for more information to generate metadata artifacts if you are using an older Integration Studio workspace.
```
Comment (samgnaniah) [inline en/docs/tutorials/integration-tutorials/service-catalog-tutorial-for-proxy-services.md:11]:
```suggestion
        For more information on generating metadata artifacts if you are using an older Integration Studio workspace, see [Generating Service Catalog Metadata Artifact]({{base_path}}/integrate/develop/generate-service-catalog-metadata.md).
```

File: en/docs/get-started/about-this-release.md [hunk 2]
```diff
@@ -65,6 +65,7 @@ It is now available to download from [here](https://wso2.com/integration/integra
 
 ### New features
 
+- **[Generating Service Catalog Metadata Artifact for Older Workspaces]({{base_path}}/integrate/develop/generate-service-catalog-metadata.md)**
```
Comment (samgnaniah) [inline en/docs/get-started/about-this-release.md:68]: Shall we add a small description as well under this?

File: en/docs/tutorials/integration-tutorials/service-catalog-tutorial.md [hunk 3]
```diff
@@ -33,6 +33,9 @@ The following concepts and artifacts are used in this tutorial:
 ## Let's get started!
 
 Follow the steps given below to build this use case and try it out.
+    !!! Note
+        See the [Generating Service Catalog Metadata Artifact]({{base_path}}/integrate/develop/generate-service-catalog-metadata.md) for more information to generate metadata artifacts if you are using an older Integration Studio workspace.
```
Comment (samgnaniah) [inline en/docs/tutorials/integration-tutorials/service-catalog-tutorial.md:37]: Same fix as above.

File: en/docs/tutorials/integration-tutorials/service-catalog-tutorial-for-proxy-services.md (hunk 1 above)
Comment (sajithaliyanage) [reply en/docs/tutorials/integration-tutorials/service-catalog-tutorial-for-proxy-services.md]: Fixed

File: en/docs/get-started/about-this-release.md (hunk 2 above)
Comment (sajithaliyanage) [reply en/docs/get-started/about-this-release.md]: Fixed

File: en/docs/tutorials/integration-tutorials/service-catalog-tutorial.md (hunk 3 above)
Comment (sajithaliyanage) [reply en/docs/tutorials/integration-tutorials/service-catalog-tutorial.md]: Fixed

File: en/docs/get-started/about-this-release.md [hunk 4]
```diff
@@ -65,6 +65,9 @@ It is now available to download from [here](https://wso2.com/integration/integra
 
 ### New features
 
+- **[Generating Service Catalog Metadata Artifacts]({{base_path}}/integrate/develop/generate-service-catalog-metadata.md)**
+
+This feature generates metadata artifacts for the APIs and Proxy services has been created inside the older Integration Studio workspaces
```
Comment (samgnaniah) [inline en/docs/get-started/about-this-release.md:70]:
```suggestion
This feature generates metadata artifacts for the APIs and Proxy services that have been created inside the older Integration Studio workspaces.
```

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**